egui/src/ui.rs

```diff
@@ -526,6 +526,14 @@ impl Ui {
         let clip_rect = self.clip_rect().intersect(rect); // Make sure we don't paint out of bounds
         Painter::new(self.ctx().clone(), self.layer_id(), clip_rect)
     }
+
+    pub fn scroll_to_here(&mut self, center_ratio: f32) {
```

Comment:
Make sure you add this to `Response` as well.  Scrolling to the top/center/bottom of a `Ui` is less useful that scrolling to a specific widget. The `struct Response` represents the place where a widget is located.